### PR TITLE
Implement Cross-chain activity sankey chart

### PR DESCRIPTION
**Pull request by valentinoConti**
### Description

This PR implements the "Cross-chain activity" sankey chart from our Figma design and other enhancements.

The chart it's implemented with native code, no libraries added, using the canvas HTML5 element and a little bit of mathematics.

The data at this moment is coming from a hardcoded JSON created by the API team with real data, with the models supposed to be as close as possible to the future real ones.

### Extra Changes Done

- Add Figma design background on webapp landing.
- Fixed secondary colours naming to match Figma wording.
- Fix VAA count chart excluding more chains than expected.

### Screenshot

https://user-images.githubusercontent.com/41705567/220192870-bb6a6af2-3deb-4b97-9b3f-c56faf9d23f8.mov

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -74,7 +61,7 @@
 body {
   margin: 0;
   color: var(--color-white);
-  background-color: var(--color-primary-800);
+  background-image: url("./assets/Background.png");
```
Comment (raop155) [inline src/index.scss:64]: @valentinoConti I don't know why we need an image for the background (2.8 Mb).
can we do it with CSS? or any other native tool?

File: src/components/molecules/CrossChainChart/Chart.tsx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    setDestinyChainsHeight(
+      Array.from(destinyChainsRef.current.children).map(item => ({
+        itemHeight: item.getBoundingClientRect().height,
+        percentage: item.getAttribute("data-percentage"),
+      })),
+    );
+  };
+
+  useEffect(() => {
+    updateChainsHeight();
```
Comment (raop155) [inline src/components/molecules/CrossChainChart/Chart.tsx:116]: Can we call `updateChainsHeight()` after line `146`? and remove from here the `useEffect`

**Review by raop155 — APPROVED** [review]

File: src/index.scss (hunk 1 above)
Comment (valentinoConti) [reply src/index.scss]: You are right! switched the image for an SVG with only 12KB, looks almost identical 👍🏼

File: src/components/molecules/CrossChainChart/Chart.tsx (hunk 2 above)
Comment (valentinoConti) [reply src/components/molecules/CrossChainChart/Chart.tsx]: Done ✅

**Pull request merged**